### PR TITLE
Update 2.4 system requirements

### PR DESCRIPTION
**Pull request by jeff-matthews**
## Purpose of this pull request

This pull request (PR) updates 2.4 system requirements.

## Affected DevDocs pages

- https://devdocs.magento.com/guides/v2.4/install-gde/system-requirements-tech.html
- https://devdocs.magento.com/guides/v2.4/install-gde/prereq/php-settings.html

whatsnew
Updated Magento [system requirements](https://devdocs.magento.com/guides/v2.4/install-gde/system-requirements-tech.html) for MySQL, PHP, and Redis.

### CODE REVIEW

File: src/guides/v2.4/install-gde/system-requirements-tech.md [hunk 1]
```diff
@@ -33,19 +33,19 @@ Upgrading the Magento applications and extensions you obtain from Magento Market
 
 ## Database
 
-MySQL 5.7.9 and 8.0
+*  MySQL 8.0 for on-premise installations
+*  MariaDB 10.4 for {{site.data.var.ece}} projects
 
-Magento is also compatible with MySQL NDB Cluster 7.4.*, MariaDB 10.2, 10.4 (recommended,) Percona 5.7, and other binary-compatible MySQL technologies.
+Magento is also compatible, but has not been tested and is not recommended, with MySQL 5.7.9, NDB Cluster 7.4.*, MariaDB 10.2, Percona 5.7, and other binary-compatible MySQL technologies.
```
Comment (chernenm) [inline src/guides/v2.4/install-gde/system-requirements-tech.md:39]: @jeff-matthews, as per previous discussion with Max Lesechko, we, probably, have to remove NDB Cluster and other binary-compartible technologies. 
Magento is also compatible, but has not been tested and is not recommended, with MySQL 5.7.9, MariaDB 10.2 and Percona 5.7.

Comment (jeff-matthews) [reply src/guides/v2.4/install-gde/system-requirements-tech.md]: Done

**Comment by jeff-matthews** [comment]
Thanks @jfrontain. That's a good point. I think I've standardized on 2.4.0.

**Comment by jeff-matthews** [comment]
running tests

**Pull request merged**